pageserver/src/layered_repository/page_versions.rs

```diff
@@ -98,6 +98,24 @@ impl PageVersions {
         }
     }
 
+    pub fn read_pv(&self, off: u64) -> Result<PageVersion> {
+        let mut reader = self.reader(off)?;
+        Ok(PageVersion::des_from(&mut reader)?)
+    }
+
+    pub fn read_blob(&self, off: u64, buf: &mut Vec<u8>) -> Result<usize> {
```

Comment:
This seems redundant with `reader`. Should we change `read_pv` to use this directly?
Also I find this naming conflicting with `read_blob` in `blob.rs`. maybe `read_pv_bytes`?